elevation_mapping/src/ElevationMapping.cpp

```diff
@@ -128,7 +133,12 @@ bool ElevationMapping::readParameters()
 
   double minUpdateRate;
   nodeHandle_.param("min_update_rate", minUpdateRate, 2.0);
-  maxNoUpdateDuration_.fromSec(1.0 / minUpdateRate);
+  if (minUpdateRate == 0.0) {
+    maxNoUpdateDuration_.fromSec(0.0);
+    ROS_WARN("Rate for publishing the map is zero.");
+  } else {
+    maxNoUpdateDuration_.fromSec(1.0 / minUpdateRate);
+  }
   ROS_ASSERT(!maxNoUpdateDuration_.isZero());
```

Comment:
If the minUpdateRate is 0.0, and you build in Debug, then we run into this assert, right? In my opinion, as this is a non-time-critical check during startup, we should even check in Release mode and throw an exception resp. terminate the node.

Reply:
Okay, does ROS_ASSERT not working in Release mode?

Also, I found that the raw map is published in the point cloud callback and also in the mapUpdateTimerCallback. If the fused map is not needed, maybe the user wants to set this rate as zero.

Reply:
I would assume so (but I am not sure).

If it is an expected use case to set it to 0, I would turn the ROS_WARN into a ROS_INFO, and remove the assertion.

Reply:
That makes sense. At least, we tried to disable the mapUpdateTimer, so I will turn this into ROS_INFO.

Reply:
I have found other ROS_WARN for fused map publish rate and visibility cleanup rate.
May I also turn these into ROS_INFO?

Reply:
Yes, that makes sense. I would also update the description of the min_update_rate in the Readme, explaining what 0 update rate means.